### PR TITLE
Fix link in `New Featured Artist: Will Stetson` article

### PR DESCRIPTION
**Pull request by Copperbug**
"https://assets.ppy.sh/artists/385/Koala/Will%20Stetson%20-%20Koala%20(Self%20Cover).oszs" -> "https://assets.ppy.sh/artists/385/Koala/Will%20Stetson%20-%20Koala.osz"

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Blooshing** [comment]
Yeah, new link seems to be the correct one. Old link sends to a 404 

Blushing20231022-1698016857

File: news/2023/2023-10-22-new-featured-artist-will-stetson.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Try out [the map from the video above](https://osu.ppy.sh/beatmapsets/2078554#os
 
 ### Will Stetson - Koala
 
-Or try mapping [the VOCALOID version](https://assets.ppy.sh/artists/385/Koala/Will%20Stetson%20-%20Koala%20(Self%20Cover).oszs)!
+Or try mapping [the VOCALOID version](https://assets.ppy.sh/artists/385/Koala/Will%20Stetson%20-%20Koala.osz)!
```
Comment (Blooshing) [inline news/2023/2023-10-22-new-featured-artist-will-stetson.md:35]:
```Suggestion
Or try mapping [the VOCALOID version](https://assets.ppy.sh/artists/385/Koala/Will%20Stetson%20-%20Koala%20(Self%20Cover).osz)!
```

**Review by Blooshing — CHANGES_REQUESTED** [review]
this seems to be the correct link?

File: news/2023/2023-10-22-new-featured-artist-will-stetson.md (hunk 1 above)
Comment (pishifat) [reply news/2023/2023-10-22-new-featured-artist-will-stetson.md:35]: op's link is right?

**Review by pishifat — APPROVED** [review]

**Pull request merged**